perception/front_vehicle_velocity_estimator/config/front_vehicle_velocity_estimator.param.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 UCI SORA LAB.
```

Comment:
[MUST]
Delete copyright as discussed at https://github.com/autowarefoundation/autoware.universe/pull/4695

Reply:
Got it. Following the guideline

> We don't add copyright notice to parameter files (.param.yaml) or schema files (.schema.json)

copyright notice should be removed from param file.